### DISCUSSION TITLE
General info on VPS setup and admin

### DISCUSSION THREAD

**Discussion by donmr** [Ysurac/openmptcprouter#2987] [Q&A] [answered]
I'm looking for some basic info on how a VPS is configured and how to access it.

I installed the VPS on top of a fresh Debian 11 install (server w/o X11 etc.).
I did use UPSTREAM.  This is to learn and to test with a local VPS before deploying...  I'm waiting for the remote router hardware so I just have the VPS at this time.

I cannot get anything in or out of the VPS over the main IPV4 interface.

Pings are blocked in and out, there is no response to "ssh -p 65222", and no response to HTTP requests.  It seems to all be there, there a a lot of virtual network interfaces, firewall rules, ...  There are listeners on several ports including 65222 and 80.

This is on an old laptop so I have a shell session.

Is the only access from an OpenMPTCPRouter installation?

**Comment by Ysurac** [comment, marked as answer]
OpenMPTCProuter VPS part is mean to be installed on a VPS/Server in a DC, configuration can be different on a personal computer.

It's possible that interface wasn't properly detected, check in ```/etc/shorewall/params.net``` if you have the correct interface with external access and if not change and restart shorewall service.

Make also sure you use UPSTREAM6 parameter and not UPSTREAM if you want to use beta release with beta script. Or no parameters on stable VPS script if you want to use current stable release.
UPSTREAM6 parameter will also update Debian 11 to Debian 12.

**Reply by donmr** [reply]
Thanks, that got me going.  Shorewall had selected the wireless interface.  When I changed to the wired one (enp9s0) things started working.  It responds to ping and ssh.  An http request connects but doesn't get any content though, should it?

I know keeping this going is a lot of work and I appreciate your quick response.

**Reply by donmr** [reply]
HTTP is apparently being blocked by the firewall, it works after a "shorewall clear".
